ReactSkia/core_modules/RSkInputEventManager.cpp

```diff
@@ -151,5 +159,18 @@ void RSkInputEventManager::sendNotificationWithEventType(std::string eventType,
 }
 #endif //TARGET_OS_TV
 
+int RSkInputEventManager::registerAddListener(std::function<void(std::string,int,rnsKeyAction, rnsKey,bool)> fcb){
+  RNS_LOG_DEBUG("[registerAddListener] ");
```

Comment:
do we need a lock between add/remove and processKey for the eventCallbackMap_ ?

Reply:
Same Question , As per my understanding Add/Remove Listener will happen from App. IF then can it be parallel